src/main/java/com/troubleshooting/model/EnvironmentsModel.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * Model object for a collection of container environment variables
+ */
+package com.troubleshooting.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class EnvironmentsModel {
```

Comment:
EnvironmentModel and Environment-s-Model can be very confusing consider EnvironmentCollectionModel?

Reply:
Like `EnvironmentCollectionModel` a whole lot better